modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/rouny.dm

```diff
@@ -51,10 +51,13 @@
 
 	owner.balloon_alert(owner, "evasive movements began")
 	playsound(owner, 'modular_skyrat/modules/xenos_skyrat_redo/sound/alien_hiss.ogg', 100, TRUE, 8, 0.9)
-	to_chat(owner, span_danger("We take evasive action, making us impossible to hit with projectiles for the next [evasion_duration/10] seconds."))
+	to_chat(owner, span_danger("We take evasive action, making us impossible to hit with projectiles for the next [evasion_duration / 10] seconds."))
 	addtimer(CALLBACK(src, .proc/evasion_deactivate), evasion_duration)
 	evade_active = TRUE
 	RegisterSignal(owner, COMSIG_PROJECTILE_ON_HIT, .proc/on_projectile_hit)
+	REMOVE_TRAIT(owner, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+	addtimer(CALLBACK(src, .proc/give_back_ventcrawl), (cooldown_time * 0.8)) //They cannot ventcrawl until 80% of the cooldown has passed (48 seconds)
+	to_chat(owner, span_warning("We will be unable to crawl through vents for the next [(cooldown_time * 0.8) / 10] seconds."))
```

Comment:
The 0.8 should be a define, to indicate what it means, as it's otherwise not super straight-forward.

Reply:
That is a good idea actually